client/app/bundles/Room/components/VoteBox.jsx

```diff
@@ -29,7 +29,7 @@ export default class VoteBox extends React.Component {
 
     this.props.addSteps({
       title: 'Deck',
-      text: 'All the points are listed in this panel. Just click on the point which you would like to vote. ⁉️ means infinite, while ☕ means you have no clue.',
+      text: 'All the points are listed in this panel. Click on a point to select your estimate. ⁉️ means "unsure" and ☕ means "I need a break".',
```

Comment:
Source: https://en.wikipedia.org/wiki/Planning_poker

> Several commercially available decks use the sequence: 0, ½, 1, 2, 3, 5, 8, 13, 20, 40, 100, and optionally a ? (unsure), an infinity symbol (this task cannot be completed) and a coffee cup (I need a break), and I will make the rest of the team tea.

Reply:
haha, you're right 😉